share/wake/lib/system/remote_cache_runner.wake

```diff
@@ -162,66 +162,79 @@ export def mkRemoteCacheRunner (rscApi: RemoteCacheApi) (hashFn: Result RunnerIn
                 else failWithError "rsc: Failed to make output directory"
 
                 def doDownload (CacheSearchOutputFile path mode blob) =
-                    rscApiGetFileBlob blob "{input.getRunnerInputDirectory}/{path}" mode
+                    rscApiGetFileBlob blob path mode
```

Comment:
We might want to write down in the RunnerInput/Plan tuple documentation that only `Command` is relative to `Directory` both `Visible/Input` and `Output` are relative to `workspace`. It seems reasonable to assume either and I don't see it written down.

Reply:
I made a comment/note on this line but it would probably be good for us to put together a high level document covering all these details/designs/limitations.

We have a bunch of smaller docs, but I don't think we have the top level doc covering all the content. Mainly because most of this was just iterating on the local cache learnings/design